lib/galaxy/web/base/interactive_environments.py

```diff
@@ -162,16 +162,19 @@ def docker_cmd(self, env_override={}):
         conf = self.get_conf_dict()
         conf.update(env_override)
         env_str = ' '.join(['-e "%s=%s"' % (key.upper(), item) for key, item in conf.items()])
-        return '%s run %s -d %s -p %s:%s -v "%s:/import/" %s' % \
+        volume_str = ' '.join(['-v "%s"' % volume for volume in volumes])
```

Comment:
Yeah - that was exactly what I was thinking - thanks a bunch for making the change!